server/src/post/post.repository.ts

```diff
@@ -32,6 +33,37 @@ export class PostRepository {
     });
   }
 
+  /**
+   * @param {number} [skip] - Starts fetching posts starting at this page
+   * @param {number} [take=10] - Returns take many posts
+   * @return {PrismaPromise<Post>}
+   */
+
+  public async getAll(skip = 0, take = 10) {
+    return this._prismaService.post.findMany({
```

Comment:
Should have a filter option to select all/pending/approved posts

Reply:
Now added